client/my-sites/site-settings/jetpack-credentials/credentials-configured/index.jsx

```diff
@@ -48,7 +48,7 @@ class CredentialsConfigured extends Component {
 		const { canAutoconfigure, mainCredentials, siteId, translate } = this.props;
 
 		const isRevoking = this.state.isRevoking;
-		const protocol = get( this.props.mainCredentials, 'protocol', 'SSH' ).toUpperCase();
+		const protocol = get( mainCredentials, 'protocol', 'SSH' ).toUpperCase();
```

Comment:
this is existing so take it with a grain of salt, but is it right to assume `SSH` if no protocol is here?

Reply:
That is correct. We are setting the "default" values here.